autoscheduler/frontend/src/components/SchedulingPage/Schedule/Schedule.tsx

```diff
@@ -399,6 +421,45 @@ const Schedule: React.FC = () => {
     </div>
   ));
 
+  // When term is chagned, fetch saved availabilities for the new term
+  React.useEffect(() => {
+    if (term) {
+      fetch(`sessions/get_saved_availabilities?term=${term}`).then((res) => (
+        res.json()
+      )).then((avails: Availability[]) => {
```

Comment:
```suggestion
      fetch(`sessions/get_saved_availabilities?term=${term}`).then(
         (res) => res.json(),
      ).then((avails: Availability[]) => {
```

Idk why I formatted this like this